language/stdlib/transaction_scripts/create_child_vasp_account.move

```diff
@@ -29,4 +46,18 @@ fun create_child_vasp_account<CoinType>(
         LibraAccount::restore_withdraw_capability(vasp_withdrawal_cap);
     };
 }
+spec fun create_child_vasp_account {
```

Comment:
Add ensures (if it's not too complex)?

Reply:
😢  Even the simplest of ensures seems to make the prove unhappy. I think once we have more specs in LibraAccount we'll be able to add some.